### ISSUE TITLE
Incorrect border painting with TableRow

### ISSUE THREAD

**Issue by Doflatango** [DavBfr/dart_pdf#571]
**To Reproduce**
Code snippet to reproduce the behavior:
```dart
pw.TableRow(
  children: [
    pw.Text('AAAA'),
    pw.Text('BBBB BBBB BBBB BBBB BBBB BBBB BBBB BBBB'),
    pw.Text('CCCC CCCC'),
    pw.Text('DDDD DDDD'),
    pw.Text('EEEE EEEE'),
  ],
  decoration: pw.BoxDecoration(
    border: pw.Border.all(),
  ),
),
```

**Screenshots**
![image](https://user-images.githubusercontent.com/3091033/108978022-20269880-76c4-11eb-874d-6d76429ea7c4.png)

**Comment by DavBfr** [comment]
Thanks, I'll look into it.

**Comment by DavBfr** [comment]
Fixed in pdf version 3.0.0.
Don't hesitate to [Buy Me A Coffee](https://www.buymeacoffee.com/JORBmbw9h).
